### PR TITLE
feat: rate-limiter

### PR DESCRIPTION
**Pull request by aruznieto**
Add Express Login Limiter

## Pull Request type

Please check the type of change your PR introduces:

- [ ] Bugfix
- [x] Feature
- [ ] Code style update (formatting, renaming)
- [ ] Refactoring (no functional changes, no API changes)
- [ ] Build-related changes
- [ ] Documentation content changes
- [ ] Other (please describe):

## What is the current behavior?

Issue Number: #175

## Does this introduce a breaking change?

- [ ] Yes
- [x] No

### CODE REVIEW

File: backend/routes/auth.js [hunk 1]
```diff
@@ -1,8 +1,18 @@
 import express from "express";
+import rateLimit from "express-rate-limit";
 const router = express.Router();
 
 import * as auth from "../services/auth.js";
 
+const loginLimiter = rateLimit({
```
Comment (dec0dOS) [inline backend/routes/auth.js:7]: You might want to consider using environment variables. Relying on any unchangeable constants in the code may not be the best choice.
Please document any new env variables in the README section.

Comment (aruznieto) [reply backend/routes/auth.js:7]: you mean that windowsMs and max must be parameterised in env variables?

Comment (dec0dOS) [reply backend/routes/auth.js:7]: Exactly! :)

Comment (dec0dOS) [reply backend/routes/auth.js:7]: Default value should also be presented. In my opinion, the current limit seems a bit too strict, which could potentially create some difficulties for users trying to come up with their own passwords

Comment (aruznieto) [reply backend/routes/auth.js:7]: And where can I setup new env variables?

Comment (dec0dOS) [reply backend/routes/auth.js:7]: Simply setting the environment variable should suffice. There are numerous examples available for reference in https://github.com/dec0dOS/zero-ui/blob/main/backend/app.js

https://github.com/dec0dOS/zero-ui/blob/9b1ac15b463f17c197be97609e9a76922026a4df/backend/app.js#L23

Comment (dec0dOS) [reply backend/routes/auth.js:7]: This statement is incorrect. You should set env variables to the ZeroUI app.
`You should write it on environment zerotier service`

Comment (aruznieto) [reply backend/routes/auth.js:7]: I mean that user should write it on `docker-compose.yml` "zerotier" service. Following ZT_ALLOW_MANAGEMENT_FROM

Comment (dec0dOS) [reply backend/routes/auth.js:7]: No, that logic is actually handled on the ZeroUI side. It doesn't really relate to the ZeroTier controller

Comment (aruznieto) [reply backend/routes/auth.js:7]: > No, that logic is actually handled on the ZeroUI side. It doesn't really relate to the ZeroTier controller

👍🏻

**Comment by aruznieto** [comment]
@dec0dOS I think all is fixed.

**Pull request merged**